tools/cluster_deployer/templates/router_bot_poller/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM python:3.6.9-slim-stretch
+
+VOLUME /logs
+WORKDIR /app
+
+ENV PYTHONPATH="${PYTHONPATH}:/app"
+
+RUN apt-get update && \
+    apt-get install -y git locales && \
+    printf '%s\n%s\n' 'en_US.UTF-8 UTF-8' 'ru_RU.UTF-8 UTF-8' >> /etc/locale.gen && \
+    locale-gen && \
+    apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false && \
+	rm -rf /var/lib/apt/lists/*
+
+ENV LANG='en_US.UTF-8' LANGUAGE='en_US.UTF-8' LC_ALL='en_US.UTF-8'
+
+RUN git clone https://github.com/deepmipt/convai_router_bot_poller.git . && \
+    git checkout stand && \
```

Comment:
Need to be removed after merging 'stand' brunch to 'main' in convai_router_bot_poller